healthcareai/advanced_supvervised_model_trainer.py

```diff
@@ -555,6 +543,7 @@ def _create_trained_supervised_model(self, algorithm, include_factor_model=True)
             test_set_class_labels=test_set_class_labels,
             test_set_actual=self.y_test,
             metric_by_name=self.metrics(algorithm),
+            class_labels=class_labels,
```

Comment:
Was this supposed to be `= self.class_labels` (like on 525 above)?

Also, the lack of whitespace around `=` doesn't match other sections...

Reply:
1. Good eye, except that is set elsewhere on 519-525.
2. PEP8 says no spaces around equal signs in function arguments.